spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactoryIntegrationTests.java

```diff
@@ -92,6 +94,9 @@ public void open() {
 
 	@After
 	public void close() {
+		if (!this.connectionFactory.getVirtualHost().equals("non-existent")) {
```

Comment:
What is the point of this check ?
Seems for me we don't use other virtual hosts, unless the default one.